bcda/api/requests.go

```diff
@@ -6,6 +6,7 @@ import (
 	"strconv"
 
 	"github.com/go-chi/chi"
+	"github.com/jackc/pgx"
```

Comment:
New dependency; might consider a security checklist to make sure we aren't introducing a dependency that might be troublesome.